jim-github.js

```diff
@@ -217,58 +217,78 @@ function createIssue(github, username, token, repository, issue, comments, miles
             }
         };
 
+        var options_1 = {
```

Comment:
I'd rename the "options_1" to "getRequestOptions" and "options" to "postRequestOptions" so that we can easily see what the requests are.  It's a bit confusing seeing "options" and "options_1" as it's now hard to tell what they are for.